packages/nocodb/src/controllers/forms.controller.ts

```diff
@@ -44,6 +45,7 @@ export class FormsController {
       body,
       tableId,
       user: req.user,
+      req,
```

Comment:
since we include `req` now, we don't need to pass `req.user` again.

Reply:
Will do the cleanup part in a follow-up pr